### ISSUE TITLE
Unable to install using Yarn

### ISSUE THREAD

**Issue by pthm** [bee-queue/arena#99]
When installing via yarn this error is thrown.

```
error bull-arena@2.4.4: The engine "yarn" is incompatible with this module. Expected version "YARN NO LONGER USED - use npm instead.".
```

As I understand it Mixmax have switched to NPM internally but this engine definition affects users of the modules when trying to install using alternative package managers.

**Comment by pthm** [comment]
Not sure if its viable as I havent tested it on a live package but I think something along these lines would allow you to keep the engine in place for local/internal development but momentarily remove it when publishing

```json
  "scripts": {
    "prepublish": "cp package.json package.json.backup && sed -i '' '/\"yarn\"/ d' package.json",
    "postpublish": "rm package.json && mv package.json.backup package.json"
  },
```

Here I am just using the stream editor to remove the line matching `'/\"yarn\"/` in the package.json before publish and then restoring it from a backup afterwards

**Comment by ttacon** [comment]
Fully agreed on this, we'll update `yarn` support very soon.

**Comment by Nemmo** [comment]
Hi, what's wrong with yarn? I have to fix version at '2.4.3'

**Comment by wearhere** [comment]
Fixed in 2.4.5 folks, sorry about the breaking change. Mixmax has indeed [moved back to npm](https://mixmax.com/blog/to-yarn-and-back-again-npm) for development (this engine requirement was introduced by the [automated tool](https://github.com/mixmaxhq/deyarn) we used for the migration), but y'all ought to be able to use Yarn to install the module if you like.

@pthm thanks for the suggestion about dynamically modifying the engine requirement but I just removed it altogether. I think it'll be easy enough to remember not to use Yarn for development.
